src/ConfigParser/ConfigParser.js

```diff
@@ -643,4 +542,76 @@ function featureToPlugin (featureElement) {
 
     return plugin;
 }
+
+class BaseResource {
+    constructor (attrs, { platform = null } = {}) {
+        // null means resource is shared between platforms
+        this.platform = platform || null;
+        this.src = attrs.src;
+        this.target = attrs.target || undefined;
+    }
```

Comment:
If you want to add a doc block for in-code for documentation purpose, you can mention that the `target` attribute/property is only for the Windows & Electron platform.